deploy/crds/aws.managed.openshift.io_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml

```diff
@@ -32,6 +32,11 @@ spec:
           - "ec2:DeleteVpnConnectionRoute"
           - "ec2:ModifyVpcPeeringConnectionOptions"
           - "ec2:RejectVpcPeeringConnection"
+          - "ec2:CreateNetworkInterface"
+          - "ec2:DescribeNetworkInterface"
+          - "ec2:ModifyNetworkInterfaceAttribute"
+          - "ec2:DeleteNetworkInterface"
+          - "ec2:CreateNetworkInterfacePermission"
```

Comment:
@dustman9000 don't you need to add  vpc:TransientSomething permissions?

Reply:
@rogbas - not that i'm aware of...do you mean "vpc:transit"? We are not using those for PrivateLink...

Reply:
ok then, just checking